darwin/DarwinProcess.c

```diff
@@ -7,10 +7,15 @@ in the source distribution for its full text.
 
 #include "darwin/DarwinProcess.h"
 
+#include <AvailabilityMacros.h>
+#if MAC_OS_X_VERSION_MIN_REQUIRED >= 1050
 #include <libproc.h>
+#endif
```

Comment:
See docs/styleguide.md. Conditional includes should be below headers that are always included.